src/client/src/notificationBootstrapper.tsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import { Trade } from 'rt-types'
+import { Themes } from 'shell/theme'
+import Theme from 'ui/theme/Theme'
 import TradeNotification from './shell/notification/TradeNotification'
 
 declare const window: any
 
 const dismissNotification = () => window.fin.desktop.Notification.getCurrent().close()
 
-const handleNotificationMessage = (message: Trade) => {
+interface Message {
+  tradeNotification: Trade
+  theme: Themes
+}
+
+const handleNotificationMessage = ({ tradeNotification, theme }: Message) => {
```

Comment:
I'm having trouble seeing where the theme is added. The value would only be `light` or `dark`, right?

Reply:
`shell/openFin/openFin.ts`

Look for openTradeNotification.